app/controllers/spree/admin/payments_controller.rb

```diff
@@ -135,7 +135,9 @@ def authorize_stripe_sca_payment
         return unless @payment.payment_method.class == Spree::Gateway::StripeSCA
 
         @payment.authorize!
-        raise Spree::Core::GatewayError, I18n.t('authorization_failure') unless @payment.pending?
+        return if @payment.pending? && @payment.cvv_response_message.nil?
+
+        raise Spree::Core::GatewayError, I18n.t('authorization_failure')
```

Comment:
This should be fine to just make the tests pass for now, I'll likely be changing this shortly as part of https://github.com/openfoodfoundation/openfoodnetwork/issues/4178 anyway 👍